processor/src/test/java/org/mapstruct/ap/test/nestedbeans/NestedSimpleBeansMappingTest.java

```diff
@@ -41,6 +44,13 @@
 @RunWith(AnnotationProcessorTestRunner.class)
 public class NestedSimpleBeansMappingTest {
 
+    @Rule
+    public final GeneratedSource generatedSource = new GeneratedSource().addComparisonToFixtureFor(
```

Comment:
Yes, I would have expected a test which accesses the state of the target object.